test/images/Makefile

```diff
@@ -12,19 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all image push clean
+REGISTRY ?= gcr.io/k8s-e2e-images
```

Comment:
@ixdy Should we go all in here?
What about the tags then -- should everything start from 1.0 or should they have their cryptic tags as-is now?